### PR TITLE
fix: low contrast text and Istio box

### LINKED ISSUE #4909: Low Text contrast in dark mode for learning paths
### Description

Currently , the text is a bit lighter in dark mode of [learning paths](https://layer5.io/learn/learning-paths/mastering-service-meshes-for-developers/introduction-to-service-meshes/istio/getting-started) which create a visiblity issue.
The istio logo is surrounded by border box.We need to remove it  as well. 
image

### Expected Behavior

All the labs of the learning path should have better visibility 

### Environment:
- Host OS:
- Browser:

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Shiv-2712**
**Description**
This PR fixes #4909 
The text in the dark mode of the [learning paths](https://layer5.io/learn/learning-paths/mastering-service-meshes-for-developers/introduction-to-service-meshes/istio/getting-started) has been improved to address the visibility issue. Additionally, the border box surrounding the Istio logo has been eliminated.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 379e75a704e17b5169b88aae7aacfb45dd544543 at: https://650575e7a043597a50680e2d--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit c50f5cb6edcf9fb6d2911d396c19d78d22e3f036 at: https://6505f867ef6fec3a97b2c454--layer5.netlify.app

**Comment by Ghat0tkach** [comment]
hey @Shiv-2712 
Let's discuss it on the websites call.
I have already added your agenda as part of today's meeting minutes

https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

File: src/theme/app/themeStyles.js [hunk 1]
```diff
@@ -381,7 +381,7 @@ export const darktheme = {
   grey313131ToYellowFFF7D8: "#313131",
   grey323232ToGreenF3FFFD: "#323232",
   grey3C3C3CToGreyF1F1F1: "#3C3C3C",
-  grey404040ToGreen3C494F: "#404040",
+  grey404040ToGreen3C494F: "#ffffff66",
```
Comment (leecalcote) [inline src/theme/app/themeStyles.js:384]: Thanks, @Shiv-2712. Where else is this color used? Can you confirm that the use of this color in other locations on the site are not negatively affected?

Comment (Shiv-2712) [reply src/theme/app/themeStyles.js:384]: Welcome! This specific color is exclusively employed in Learn-Components, specifically in the file Chapter Style (chapter.style.js). No other file utilizes this particular color variable.

**Comment by abhijeetgauravm** [comment]
> **Description** The text in the dark mode of the [learning paths](https://layer5.io/learn/learning-paths/mastering-service-meshes-for-developers/introduction-to-service-meshes/istio/getting-started) has been improved to address the visibility issue. Additionally, the border box surrounding the Istio logo has been eliminated.
> 
> **[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
> 
> * [x]  Yes, I signed my commits.

hey @Shiv-2712 it would be good if you just edit description with like This PR fixes #issue_no

**Comment by l5io** [comment]
🚀 Preview for commit 87c7594abde2bcc4d927fb8dc75373fb97572ab4 at: https://651c1ffb7992d33137a8eeaf--layer5.netlify.app

**Comment by Mohith234** [comment]
@Shiv-2712 Update the PR description with correct mention of the issue

**Comment by abhijeetgauravm** [comment]
> **Description** This PR fixes #4922 The text in the dark mode of the [learning paths](https://layer5.io/learn/learning-paths/mastering-service-meshes-for-developers/introduction-to-service-meshes/istio/getting-started) has been improved to address the visibility issue. Additionally, the border box surrounding the Istio logo has been eliminated.
> 
> **[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
> 
> * [x]  Yes, I signed my commits.

hey @Shiv-2712 Please mention the issue you are fixing. Here u are fixing #4909. So u should edit the PR description with this.

**Comment by abhijeetgauravm** [comment]
Hey @Shiv-2712 Let's discuss this on website call today at 5:30 PM IST / 7 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima)

**Review by leecalcote — APPROVED** [review]
Thanks!

**Pull request merged**